programs/svm-spoke/src/instructions/handle_receive_message.rs

```diff
@@ -11,9 +11,6 @@ use crate::{
     State,
 };
 
-//TODO: we have inconsistent imports in this file, in some places referencing from source crates (SvmSpoke::id)
```

Comment:
we have not addressed this

Reply:
right but the thing it was referencing for some reason could not be refactored like other imports.